cmd/provider/main.go

```diff
@@ -91,8 +93,8 @@ func main() {
 		Options: xpcontroller.Options{
 			Logger:                  log,
 			GlobalRateLimiter:       ratelimiter.NewGlobal(*maxReconcileRate),
-			PollInterval:            1 * time.Minute,
-			MaxConcurrentReconciles: 1,
+			PollInterval:            *pollInterval,
+			MaxConcurrentReconciles: *maxReconcileRate,
```

Comment:
Should we have separate parameters for the max reconcile rate (which is a global rate shared between all reconcilers) and the max concurrent reconciles (which is a count of workers consuming the workqueue of a controller)?

Reply:
@ulucinar I would like to be consistent with other providers and @negz has a detailed description [here](https://github.com/crossplane/crossplane/issues/2595) on why it makes sense to use the same parameter for both.

Reply:
Thanks for the pointer @turkenh. I agree, we had better follow the convention suggested there.